src/core/common/settings.hpp

```diff
@@ -123,9 +123,10 @@ class SettingsBase : public InstanceLocator
         kKeySrpClientInfo     = OT_SETTINGS_KEY_SRP_CLIENT_INFO,
         kKeySrpServerInfo     = OT_SETTINGS_KEY_SRP_SERVER_INFO,
         kKeyNat64Prefix       = OT_SETTINGS_KEY_NAT64_PREFIX,
+        kKeyBrUlaPrefix       = OT_SETTINGS_KEY_BR_ULA_PREFIX,
```

Comment:
Suggest to rename `kKeyOmrPrefix` to `kKeyBrUlaPrefix` and remove `kKeyNat64Prefix`. 
I think we only need to save BR ULA prefix?

Reply:
Updated.